libs/wingc/src/jsify.rs

```diff
@@ -1083,26 +1079,8 @@ impl<'a> JSifier<'a> {
 				code.add_code(self.jsify_scope_body(scope, ctx));
 				code
 			}
-			FunctionBody::External(external_spec) => {
-				debug!(
-					"Resolving extern \"{}\" from \"{}\"",
-					external_spec, self.absolute_project_root
-				);
-				let resolved_path =
-					match wingii::node_resolve::resolve_from(&external_spec, Utf8Path::new(&self.absolute_project_root)) {
-						Ok(resolved_path) => resolved_path.to_string().replace("\\", "/"),
-						Err(err) => {
-							report_diagnostic(Diagnostic {
-								message: format!("Failed to resolve extern \"{external_spec}\": {err}"),
-								span: Some(func_def.span.clone()),
-								annotations: vec![],
-							});
-							format!("/* unresolved: \"{external_spec}\" */")
-						}
-					};
-				CodeMaker::one_line(format!(
-					"return (require(\"{resolved_path}\")[\"{name}\"])({parameters})"
-				))
+			FunctionBody::External(file_path) => {
+				CodeMaker::one_line(format!("return (require(\"{file_path}\")[\"{name}\"])({parameters})"))
```

Comment:
Need to make this PR a breaking change and update specs/docs that mention the ability to use extern with node modules directly.

Reply:
Done